authentication/native/server.go

```diff
@@ -117,3 +126,17 @@ func (server *authServer) validateSignature(token authentication.AuthToken) erro
 func (server *authServer) IsInterfaceNil() bool {
 	return server == nil
 }
+
+func (server *authServer) getBlockByHash(ctx context.Context, hash string) (*data.Block, error) {
+	var block data.Block
+	buff, code, err := server.httpClientWrapper.GetHTTP(ctx, server.apiNetworkAddress+"/blocks/"+hash)
```

Comment:
extract "/blocks/" in a constant then the full path concatenation in a local var

Reply:
done